src/main/resources/init_db.sql

```diff
@@ -1,9 +1,18 @@
-CREATE SCHEMA IF NOT EXISTS `taxi_service` DEFAULT CHARACTER SET utf8;
-USE `taxi_service`;
+CREATE DATABASE `tax_service_db` /*!40100 DEFAULT CHARACTER SET utf8mb3 */ /*!80016 DEFAULT ENCRYPTION='N' */;
 
 CREATE TABLE `manufacturers` (
-                                        `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                        `name` VARCHAR(225) NOT NULL,
-                                        `country` VARCHAR(225) NOT NULL,
-                                        `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                        PRIMARY KEY (`id`));
+                                 `name` varchar(255) DEFAULT NULL,
+                                 `country` varchar(255) DEFAULT NULL,
+                                 `id` bigint NOT NULL AUTO_INCREMENT,
+                                 `is_deleted` tinyint DEFAULT '0',
+                                 PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3;
```

Comment:
```suggestion
);
```
It's default value so can be dropped